src/lib/components/addons/AddOnsNavigation.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+  import { _ } from "svelte-i18n";
+  import { Infinity16, StarFill16 } from "svelte-octicons";
+
+  import Pin from "@/common/icons/Pin.svelte";
+  import Premium from "@/common/icons/Premium.svelte";
+  import SidebarItem from "$lib/components/sidebar/SidebarItem.svelte";
+  import Flex from "../common/Flex.svelte";
+
+  import { CATEGORIES } from "$lib/api/addons";
+
+  export let active: string = "all";
+</script>
+
+<Flex direction="column" gap={1}>
+  <header class="header">
+    <h2>{$_("addonBrowserDialog.title")}</h2>
+    <p>{$_("addonBrowserDialog.subtitle")}</p>
+  </header>
+  <Flex direction="column">
+    <SidebarItem
+      active={active === "all"}
+      href="/app/add-ons"
+      --hover="var(--blue-2)"
```

Comment:
These hover colors aren't coming through in the preview link. Maybe something's not connected?